catalogue/webapp/package.json

```diff
@@ -27,22 +27,22 @@
     "agentkeepalive": "^4.2.1",
     "babel-core": "^7.0.0-bridge.0",
     "babel-jest": "^24.5.0",
+    "graphql": "^16.6.0",
```

Comment:
Are we using these `graphql` dependencies?

Reply:
I have no clue, I installed react-photo-album and npm noticed that the package.json wasn't alphabetical and corrected it

Reply:
we use `graphql-request` in the new search for Prismic content, `graphql` [is a dependency](https://www.npmjs.com/package/graphql-request#install).